values.yaml

```diff
@@ -89,3 +89,20 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+postgresql:
+  enabled: true
+  primary:
+    initdb:
+      scripts:
+        create-db.sql:
+          CREATE DATABASE grader_config;
+        create-schema.sql: |
+          \connect grader_config;
```

Comment:
is that an errant `\`? or does the script need it for some reason?

Reply:
the `\` is needed